crates/cli/src/app.rs

```diff
@@ -186,6 +186,12 @@ pub enum Commands {
     )]
     Teardown,
 
+    #[command(
+        name = "visualize",
+        alias = "vis",
+        about = "Start an interactive monorepo visualization graph"
+    )]
+    Visualize,
```

Comment:
The visualizer would be for `moon dep-graph` and `moon project-graph`, we shouldn't add a 3rd command.

Reply:
Alright makes sense. I thought those were reserved for the dot versions.